gcn/notices/chime/detection.example.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/chime/frb.schema.json",
+  "alert_type": "detection",
```

Comment:
This is not a valid option, the allowed values are:
```json
"allowedValues": [
        "initial",
        "update",
        "retraction"
      ]

```

These are defined here https://github.com/nasa-gcn/gcn-schema/blob/main/gcn/notices/core/Alert.schema.json

Reply:
@tabbott36 we are working on adding subsequent (see https://github.com/nasa-gcn/gcn-schema/pull/221). 
However, initial will like remain same, else we need to take votes from the missions which are already using it.

For validation, as detection and subsequent please add a new property, current alert_type is not consistent.

Reply:
Ok, sounds good to me. Thanks, Vidushi!